arches_controlled_lists/models.py

```diff
@@ -219,6 +224,29 @@ def build_tile_value(self):
         }
         return tile_value
 
+    def build_select_option(self):
+        labels = getattr(self, "list_item_labels", self.list_item_values.labels())
+        ranked_labels = sorted(
+            labels,
+            key=lambda label: rank_label(
+                kind=label.valuetype_id,
+                source_lang=label.language_id,
+                target_lang=translation.get_language(),
+            ),
+            reverse=True,
+        )
```

Comment:
nit: since this is being used in several places, it could be abstracted to its own function